### PR TITLE
Update jekyll-preview.yml

### PR DESCRIPTION
**Pull request by delisma**
Added Gemfile

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+group :jekyll_plugins do
+  gem "jekyll-remote-theme", github: "benbalter/jekyll-remote-theme", branch: "master"
```
Comment (nschonni) [inline Gemfile]: Don't think you need the explicit reference to the gem, as I believe it's part of the default https://github.com/github/pages-gem/
I think you just need to set it in the `_config.yml` https://stackoverflow.com/questions/49116592/customize-jekyll-remote-theme-for-github-pages

Comment (delisma) [reply Gemfile]: I think so too, but I'm trying to trigger the GitHub Action to build Jekyll and didn't know what should be in the Gemfile.

Comment (nschonni) [reply Gemfile]: Gotcha, might want to do the `github-pages` gem then, like https://github.com/canada-ca/ore-ero/blob/master/Gemfile

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1 @@
+source "https://rubygems.org"
```
Comment (nschonni) [inline Gemfile]: You'd need to regenrate or ignore the Gemfile.lock too
```suggestion
source "https://rubygems.org"

gem 'github-pages'
```

**Pull request merged**